app/code/Magento/CatalogImportExport/Model/Import/Product.php

```diff
@@ -2728,6 +2741,9 @@ private function parseAttributesWithWrappedValues($attributesData)
      */
     public function parseMultiselectValues($values, $delimiter = self::PSEUDO_MULTI_LINE_SEPARATOR)
     {
+        if ($delimiter !== $this->getMultipleLineSeparator()) {
```

Comment:
this is a bit confusing as it makes the delimiter parameter obsolete.
Also 3rd party integrations could rely on being able to pass in another delimiter.

It would also change the behavior of \Magento\CatalogImportExport\Model\Import\Product::parseMultipleValues (if this change is intended, I would suggest changing the parseMultipleValues function to inject the correct parameter and removing this condition here in parseMultiselectValues)

Reply:
Thanks for the suggestion, i didn't notice it, i will update the PR.